tests/providers/amazon/aws/waiters/test_custom_waiters.py

```diff
@@ -424,3 +425,42 @@ def test_steps_failed(self, mock_list_steps):
                 StepIds=[self.STEP_ID1, self.STEP_ID2],
                 WaiterConfig={"Delay": 0.01, "MaxAttempts": 3},
             )
+
+
+class TestCustomDataBrewWaiters:
+    """Test waiters from ``amazon/aws/waiters/glue.json``."""
+
+    JOB_NAME = "test_job"
+    RUN_ID = "123"
+
+    @pytest.fixture(autouse=True)
+    def setup_test_cases(self, monkeypatch):
+        self.client = boto3.client("databrew", region_name="eu-west-3")
+        monkeypatch.setattr(GlueDataBrewHook, "conn", self.client)
+
+    def test_service_waiters(self):
+        hook_waiters = GlueDataBrewHook(aws_conn_id=None).list_waiters()
+        assert "job_complete" in hook_waiters
+
+    @pytest.fixture
+    def mock_describe_job_runs(self):
+        """Mock ``GlueDataBrewHook.Client.describe_job_run`` method."""
+        with mock.patch.object(self.client, "describe_job_run") as m:
+            yield m
+
+    @staticmethod
+    def describe_jobs(status: str):
+        """
+        Helper function for generate minimal DescribeJobRun response for a single job.
+        https://docs.aws.amazon.com/databrew/latest/dg/API_DescribeJobRun.html
```

Comment:
I know it's a nitpick, but an upcoming pydocstyle rule enforces a newline there so we may as well do it now.

```suggestion
        Helper function for generate minimal DescribeJobRun response for a single job.
        
        https://docs.aws.amazon.com/databrew/latest/dg/API_DescribeJobRun.html
```